airflow/models.py

```diff
@@ -3732,6 +3732,32 @@ def latest_execution_date(self, session=None):
         ).scalar()
         return execution_date
 
+    @property
+    @provide_session
+    def schedulable_at(self, session=None):
+        """
+        Returns the earliest time at which the DAG will next be eligible for
+        execution by the scheduler
+        """
+        from airflow.jobs import SchedulerJob
+        scheduler = SchedulerJob()
+        next_run_date = scheduler.create_dag_run(self, dry_run=True)
```

Comment:
I am still getting confused here. You might have already talked about this to @ashb but it would be good if you can clear my doubt.

My confusion is the same that Ash mentioned in one of the comments, what is the difference between `schedulable_at` and `next_execution_date`?

From the code will it not give the `schedulable_at=next_execution_date + schedule_interval` ?